R/zzz.R

```diff
@@ -0,0 +1,14 @@
+# nocov start
+
+.onLoad <- function(libname, pkgname) {
+    load_renv_env()
+    renv$renv_platform_init()
+    renv$renv_envvars_init()
+    renv$renv_log_init()
+    renv$renv_methods_init()
+    renv$renv_filebacked_init()
+    renv$renv_libpaths_init()
+    renv$renv_patch_init()
```

Comment:
Does this seem right? Do I need all of it? If I did not include this, I would errors like this when trying to use functions from the renv environment:

```
Error in get(scope, envir = `_renv_filebacked`) : 
object 'settings' not found
```

Reply:
Why doesn't packrat need this?

Reply:
I took all this out, because it was resulting in the package not building. There are other problems now, but this seems like it was a bad idea.